### PR TITLE
Maven project with JUnit

### PR DESCRIPTION
**Pull request by IrinaBogatko95**

### CODE REVIEW

File: Task_11_Test/src/main/java/Movement/CarTrip.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    }
+
+    /**
+     * Returns time of trip on car
+     * @param checkpoints list of all checkpoints of trip
+     */
+    public double getTripTime(ArrayList<Checkpoint> checkpoints) {
+        Distance distance = new Distance();
+        double time = distance.getDistance(checkpoints) / SPEED;
+        return 1000 * time / 1000;
```
Comment (AlexanderSoroka) [inline Task_11_Test/src/main/java/Movement/CarTrip.java]: 1000 \* time / 1000 - does it just equal to time?

File: Task_11_Test/src/main/java/Movement/GetCheckpoints.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ */
+public class GetCheckpoints {
+
+    /**
+     * Read checkpoints from file and add them in list
+     */
+    public ArrayList<Checkpoint> readCheckpoints() {
+        ArrayList<Checkpoint> list = new ArrayList<Checkpoint>();
+        try {
+            BufferedReader br = new BufferedReader(new FileReader("E:\\points.txt"));
```
Comment (AlexanderSoroka) [inline Task_11_Test/src/main/java/Movement/GetCheckpoints.java]: do not use hardcoded paths in the code - pass it as method argument

File: Task_11_Test/src/test/java/Movement/BicycleTripTest.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    @Test
+    public void testGetTripTimeNotNegativeNumbers() throws Exception {
+        Checkpoint checkpoint1 = new Checkpoint(0.0, 0.0);
+        Checkpoint checkpoint2 = new Checkpoint(-100.0, 0.0);
+        Checkpoint checkpoint3 = new Checkpoint(-57.0, 0.0);
+        checkpoints.add(checkpoint1);
+        checkpoints.add(checkpoint2);
+        checkpoints.add(checkpoint3);
+        double time = bicycleTrip.getTripTime(checkpoints);
+        assertNotEquals(-143, time);
```
Comment (AlexanderSoroka) [inline Task_11_Test/src/test/java/Movement/BicycleTripTest.java]: does it mean -142 would be ok?

File: Task_11_Test/src/test/java/Movement/DistanceTest.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+    @Test
+    public void testGetDistanceNotNegative() {
+        Checkpoint checkpoint1 = new Checkpoint(0.0, 0);
+        Checkpoint checkpoint2 = new Checkpoint(-100.0, 0);
+        Checkpoint checkpoint3 = new Checkpoint(-40.0, 0);
+        checkpoints.add(checkpoint1);
+        checkpoints.add(checkpoint2);
+        checkpoints.add(checkpoint3);
+        assertNotEquals(-60, distance.getDistance(checkpoints));
```
Comment (AlexanderSoroka) [inline Task_11_Test/src/test/java/Movement/DistanceTest.java]: I suppose -59 is ok. Please comment why you decided to use NotEquals here in so strange manner

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

File: Task_11_Test/src/test/java/Movement/DistanceTest.java (hunk 4 above)
Comment (IrinaBogatko95) [reply Task_11_Test/src/test/java/Movement/DistanceTest.java]: You are right, this test is useless.

**Pull request merged**